team-builder/src/components/Form.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+
+const Form = (props) => {
+  const [member, setMember] = useState([{name: "", email: "", role: ""}]);
+
+  const handleChange = event => {
+    console.log(member);
+    setMember({
+      ...member, [event.target.name]: event.target.value
+    });
+  };
+
+    const submitHandler = event => {
+      event.preventDefault();
+      // console.log(username)
```

Comment:
your overall logic here is good, just need to pass the data somewhere once the submit is done. right now you should be receiving data but its not doing anything when you press the submit handler. also comments are nice =]